src/components/doctors/Doctors.jsx

```diff
@@ -58,7 +58,7 @@ const Doctors = () => {
             <div className={styles.image_container}>
               <img className={styles.img} src={doctor.image_url} alt={doctor.name} />
             </div>
-            <p className={styles.name}>{doctor.name}</p>
+            <div className={styles.name}><NavLink to={`/doctors/${doctor.id}`}>{doctor.name}</NavLink></div>
             <p className={styles.specialization}>{doctor.specialization}</p>
             <ul className={styles.social_icons}>
               <li>
```

Comment:
I am asking myself what this is still doing in your codebase @mahdinoori2000.
I think you can remove it